packages/muelu/src/Graph/UncoupledAggregation/MueLu_AggregationPhase1Algorithm_kokkos_def.hpp

```diff
@@ -109,19 +100,35 @@ namespace MueLu {
     //can only enforce max aggregate size
     if(algorithm == Algorithm::Distance2)
     {
-      BuildAggregatesDistance2(graph, aggregates, aggStat,
-                               numNonAggregatedNodes, maxNodesPerAggregate);
+      if(params.get<bool>("aggregation: deterministic"))
+      {
+        std::cout << "Using deterministic aggregation for phase 1" << std::endl;
```

Comment:
Every process is going to print ....

Reply:
Yep, that was a debug statement that is now removed.